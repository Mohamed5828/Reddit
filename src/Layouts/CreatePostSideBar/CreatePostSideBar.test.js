import { shallow } from "enzyme";

// import Components
import CreatePostSideBar from "./CreatePostSideBar";
import { BrowserRouter, Routes } from "react-router-dom";

describe("Create Post Component", () => {
  it("this is a test for side-bar post Component", () => {
    expect(
      shallow(
        <BrowserRouter>
          <Routes>
            <CreatePostSideBar />
          </Routes>
        </BrowserRouter>
      )
    ).toMatchSnapshot();
  });
});
