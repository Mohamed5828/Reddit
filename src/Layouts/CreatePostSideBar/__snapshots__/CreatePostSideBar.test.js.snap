// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Post Component this is a test for side-bar post Component 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "default",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "createHref": [Function],
      "go": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "default",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Routes>
    <CreatePostSideBar />
  </Routes>
</Router>
`;
